### PR TITLE
OS region setting requirements for TDVT test

### PR DESCRIPTION
**Pull request by tabBhargav**
**Things To Remember:** 

1. Be sure to merge into the right branch. The choices are:
* **`master`** if your PR runs on the current version of Tableau.
* **`dev-2021.2`**  
2. You should update at least one of the following:
* The changelog of the project you modified.
* The root `README.md` if the PR is a release of the SDK, Connector Packager, or TDVT.  
=====

### CODE REVIEW

**Comment by rosswbrown** [comment]
It is the TDVT expected files that require that format correct?

Also I'm not sure documentation should recommend modifying the test date.

I noticed there is a bit of a troubleshooting [section](https://tableau.github.io/connector-plugin-sdk/docs/tdvt-test-case#triaging-a-specific-tdvt-test-case) here too, but this file seems to be the primary location.

File: docs/tdvt.md [hunk 1]
```diff
@@ -544,3 +544,12 @@ Check that any <span style="font-family: courier new">date-format</span> element
 
 __The agg.countd expression test and the join.null.int logical tests are failing__
 Check that your database is correctly returning column nullability information in the metadata. See [Design Considerations]({{ site.baseurl }}/docs/design) for more information.
+
+__The date string format vary according to OS date format settings__
+TDVT requires the OS locale be set to English/US so that the dates are formatted mm/dd/yyyy. 
+Another workaround is to check that all tests should store input dates (times and datetimes) in the save format as CSV has. Otherwise the format should be hardcoded to match the CSV or, better, defined somehow in test specs, for example:
```
Comment (lukewrites) [inline docs/tdvt.md:550]: Let's leave out t he below. Just say what's in line 549

**Review by lukewrites — CHANGES_REQUESTED** [review]

File: docs/tdvt.md [hunk 2]
```diff
@@ -544,3 +544,6 @@ Check that any <span style="font-family: courier new">date-format</span> element
 
 __The agg.countd expression test and the join.null.int logical tests are failing__
 Check that your database is correctly returning column nullability information in the metadata. See [Design Considerations]({{ site.baseurl }}/docs/design) for more information.
+
+__The date string format vary according to OS date format settings__
```
Comment (lukewrites) [inline docs/tdvt.md:548]:
```suggestion
__Date string format varies according to OS date format settings__
```

**Review by lukewrites — APPROVED** [review]
🚢

**Pull request merged**